### PR TITLE
Adding ability to create an App Runner service from an ECR tag

### PR DESCRIPTION
**Pull request by kiiadi**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Description

Add the ability to start the App Runner "Creation" flow from an existing ECR tag

## Motivation and Context

Copying and pasting is lame.

## Related Issue(s)

#2625 

## Testing

New unit test added

## Screenshots (if appropriate)
app-runner-from-ecr

## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/ecr/actions/CreateAppRunnerServiceAction.kt [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    SingleExplorerNodeAction<EcrTagNode>(message("ecr.create.app_runner_service.action"), null, null),
+    DumbAware,
+    CoroutineScope by ApplicationThreadPoolScope("AppRunnerCreateServiceAction") {
+    override fun actionPerformed(selected: EcrTagNode, e: AnActionEvent) {
+        val project = e.getRequiredData(PlatformDataKeys.PROJECT)
+        launch {
+            withContext(getCoroutineUiContext()) {
+                CreationDialog(project, "${selected.repository.repositoryUri}:${selected.tag}").show()
+            }
+        }
```
Comment (abrooksv) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/services/ecr/actions/CreateAppRunnerServiceAction.kt:29]: This is already on the UI thread, so the corotuine is not needed

Comment (kiiadi) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/services/ecr/actions/CreateAppRunnerServiceAction.kt:29]: It's modeled off `software.aws.toolkits.jetbrains.services.apprunner.actions.CreateServiceAction`

Comment (abrooksv) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/services/ecr/actions/CreateAppRunnerServiceAction.kt:29]: Huh...that one is wrong too :/

Comment (kiiadi) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/services/ecr/actions/CreateAppRunnerServiceAction.kt]: moved both

**Review by abrooksv — APPROVED** [review]

**Pull request merged**